### PR TITLE
liuliu/branch-protection-selector-fix

### PR DESCRIPTION
**Pull request by liuliu-dev**

### CODE REVIEW

File: cypress/e2e/dolthub/privatePaths/render/databaseSettingsPage/branchProtection.spec.ts [hunk 1]
```diff
@@ -53,14 +52,13 @@ describe(`${pageName} renders branch protection settings`, () => {
         newClickFlow(
           "[data-cy=branch-protection-branch-selector]>div>div",
           [
-            newExpectationWithSelector(
+            newExpectation(
               "should select the branch_can_not_be_deleted branch",
-              "[data-cy=branch-protection-branch-selector]>div>div>div>div>div>div>div",
-              3,
+              "[data-cy=branch_can_not_be_deleted",
```
Comment (ElTragon) [inline cypress/e2e/dolthub/privatePaths/render/databaseSettingsPage/branchProtection.spec.ts:57]: Do you mean
` "[data-cy=branch_can_not_be_deleted]",`
I think your missing a `]`

File: cypress/e2e/dolthub/privatePaths/render/databaseSettingsPage/branchProtection.spec.ts [hunk 2]
```diff
@@ -85,14 +83,13 @@ describe(`${pageName} renders branch protection settings`, () => {
         newClickFlow(
           "[data-cy=branch-protection-branch-selector]>div>div",
           [
-            newExpectationWithSelector(
+            newExpectation(
               "should select the branch_can_not_be_deleted branch",
-              "[data-cy=branch-protection-branch-selector]>div>div>div>div>div>div>div",
-              4,
+              "[data-cy=branch_can_not_be_force_pushed",
```
Comment (ElTragon) [inline cypress/e2e/dolthub/privatePaths/render/databaseSettingsPage/branchProtection.spec.ts:88]: Same are you missing a ']'

File: cypress/e2e/dolthub/privatePaths/render/databaseSettingsPage/branchProtection.spec.ts [hunk 3]
```diff
@@ -85,14 +83,13 @@ describe(`${pageName} renders branch protection settings`, () => {
...
               beVisibleAndContain("branch_can_not_be_force_pushed"),
             ),
           ],
-          "",
+          "[data-cy=branch_can_not_be_force_pushed",
```
Comment (ElTragon) [inline cypress/e2e/dolthub/privatePaths/render/databaseSettingsPage/branchProtection.spec.ts:92]: Same are you missing a ']'

**Review by ElTragon — APPROVED** [review]
lgtm

**Review by tbantle22 — APPROVED** [review]

**Pull request merged**